src/settings/base.py

```diff
@@ -239,6 +239,11 @@
         '%B %d, %Y',
     )
 }
+REST_FRAMEWORK_EXTENSIONS = {
+    'DEFAULT_CACHE_RESPONSE_TIMEOUT': 60 * 5,
+    'DEFAULT_OBJECT_CACHE_KEY_FUNC': 'rest_framework_extensions.utils.default_object_cache_key_func',
+    'DEFAULT_LIST_CACHE_KEY_FUNC': 'rest_framework_extensions.utils.default_list_cache_key_func',
```

Comment:
i don't think we need these?

Reply:
`DEFAULT_CACHE_RESPONSE_TIMEOUT` default is none. Which means the cache would never be cleared. The other two I copied from the documentation. Might be unnecessary.